test/e2e/framework/util.go

```diff
@@ -1377,7 +1377,9 @@ retriesLoop:
 		// NOTE the test may need access to the events to see what's going on, such as a change in status
 		actualWatchEvents := scenario(resourceWatch)
 		errs := sets.NewString()
-		ExpectEqual(len(expectedWatchEvents) <= len(actualWatchEvents), true, "Error: actual watch events amount (%d) must be greater than or equal to expected watch events amount (%d)", len(actualWatchEvents), len(expectedWatchEvents))
+		if (!(len(expectedWatchEvents) <= len(actualWatchEvents)) {
```

Comment:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/105939/pull-kubernetes-verify/1453650290241703936
```
test/e2e/framework/util.go:1380:61: syntax error: unexpected {, expecting )
test/e2e/framework/util.go:1381:4: syntax error: unexpected framework, expecting {
test/e2e/framework/util.go:1382:3: syntax error: unexpected }, expecting expression
```
one more `)` seems necesary after `actualWatchEvents)` for `!(`.
Or just `if (len(expectedWatchEvents) > len(actualWatchEvents)) {`

Reply:
... and drop the unnecessary brackets. This is Go, not C :grin: